IBPSA/ThermalZones/ISO13790/Validation/BESTEST/Cases6xx/Case600.mo

```diff
@@ -24,18 +25,14 @@ model Case600 "Case 600FF, but with dual-setpoint for heating and cooling"
         origin={0,72},
         extent={{-6,-6},{6,6}},
         rotation=0)));
-  Modelica.Blocks.Sources.Constant TSetHea(k=273.15 + 20) "Set-point for heating"
+  Modelica.Blocks.Sources.CombiTimeTable
```

Comment:
I took inspiration from `Buildings.ThermalZones.Detailed.Validation.BESTEST`. But it is useful because in this way you can extend the Case600 when developing for example Case640 and Case650. Here the schedule varies during the day.